lmdeploy/turbomind/deploy/converter.py

```diff
@@ -113,6 +114,50 @@ def copy_tokenizer(model_path: str, tokenizer_path: str,
                     osp.join(triton_models_path, 'tokenizer'))
 
 
+def update_output_format(model_name: str, model_format: str, model_path: str,
+                         output_format: str):
+    """Update output format according to model info."""
+    TORCH_DTYPE_MAP = {torch.bfloat16: 'bf16'}
+    MODEL_NAME_MAP = {'qwen': 'bf16'}
+    model_name = model_name.split('-')[0]
+
+    def _infer_output_format(config):
+        """_infer_output_format."""
+        torch_dtype = getattr(config, 'torch_dtype', None)
+        if torch_dtype:
+            updated_output_format = TORCH_DTYPE_MAP.get(
+                torch_dtype, output_format)
+        else:
+            # get model name prefix
+            updated_output_format = MODEL_NAME_MAP.get(model_name,
+                                                       output_format)
+        if updated_output_format == 'bf16':
+            if not torch.cuda.is_bf16_supported():
```

Comment:
I tried to convert qwen-7b-chat model in v100. The `weight_type` is supposed to be `fp16`, right? but I got `bf16`. And it really can be inferred, very confusing